person.rb

```diff
@@ -1,14 +1,21 @@
-class Person
+require './nameable'
+
+class Person < Nameable
   attr_accessor :name, :age
   attr_reader :id
 
   def initialize(age, name = 'unknown', parent_permission: true)
+    super()
     @id = Random.rnad(1..1000)
```

Comment:
- [ ] Kindly fix the typo here, it should be `rand` not `rnad` 😅

Reply:
Thank you so much for your observation and feedback. I implement your recommended fix.